test/OrchardCore.Tests/Apis/Lucene/LuceneContext.cs

```diff
@@ -0,0 +1,21 @@
+using Microsoft.Extensions.DependencyInjection;
+using OrchardCore.Environment.Shell;
+using OrchardCore.Tests.Apis.Context;
+
+namespace OrchardCore.Tests.Apis.Lucene
+{
+    public class LuceneContext : SiteContext
+    {
+        public static IShellHost ShellHost { get; }
+
+        static LuceneContext()
+        {
+            ShellHost = Site.Services.GetRequiredService<IShellHost>();
+        }
+
+        public LuceneContext()
+        {
+            this.WithRecipe("luceneQueryTest");
```

Comment:
```suggestion
           WithRecipe("luceneQueryTest");
```

Reply:
this doesn't work
![image](https://user-images.githubusercontent.com/8073622/111201231-7c415600-85ba-11eb-86a3-8cc32b2dee48.png)
AgencyContext also uses "this"

Reply:
May be because it's an extension method